packages/@aws-cdk/aws-appsync/lib/schema-intermediate.ts

```diff
@@ -47,13 +47,27 @@ export class InterfaceType implements IIntermediateType {
    * @default - no directives
    */
   public readonly directives?: Directive[];
+  /**
+   * the authorization modes for this intermediate type
+   */
+  protected modes?: AuthorizationType[];
 
   public constructor(name: string, props: IntermediateTypeProps) {
     this.name = name;
     this.definition = props.definition;
     this.directives = props.directives;
   }
 
+  /**
+   * Bind this Interface Type to a GraphQL Api
```

Comment:
update this comment to include when it's called (see bindToCluster, bindToInstance for examples)